lib/i18n/backend/active_record.rb

```diff
@@ -4,9 +4,24 @@
 module I18n
   module Backend
     class ActiveRecord
-      autoload :Missing,     'i18n/backend/active_record/missing'
-      autoload :StoreProcs,  'i18n/backend/active_record/store_procs'
-      autoload :Translation, 'i18n/backend/active_record/translation'
+      autoload :Missing,       'i18n/backend/active_record/missing'
+      autoload :StoreProcs,    'i18n/backend/active_record/store_procs'
+      autoload :Translation,   'i18n/backend/active_record/translation'
+      autoload :Configuration, 'i18n/backend/active_record/configuration'
+
+      class << self
+        def configure(&block)
+          yield(config) if block_given?
+        end
+
+        def config
+          @config ||= Configuration.new
+        end
+
+        def config=(x)
```

Comment:
@mpataki Ok, i got the problem and i think i know how to solve it.
So let's remove `config=` from here and to reset configuration in `teardown` we can use
```ruby
I18n::Backend::ActiveRecord.instance_variable_set :@config, I18n::Backend::ActiveRecord::Configuration.new
```